Formula/wso2ei-6.4.0.rb

```diff
@@ -0,0 +1,35 @@
+class Wso2ei640 < Formula
+  PRODUCT_NAME = "wso2ei".freeze
+  PRODUCT_VERSION = "6.4.0".freeze
+  PROFILES = %w[integrator business-process broker analytics-dashboard analytics-worker micro-integrator msf4j].freeze
+
+  desc "WSO2 Enterprise Integrator 6.4.0"
+  homepage "https://wso2.com/integration/"
+  url "https://bintray.com/wso2/binary/download_file?file_path=#{PRODUCT_NAME}-#{PRODUCT_VERSION}.zip"
+  sha256 "36ab3982cc517674ab1673c1947093bf8022e1f4fb971a2fe6fa1b3085a13917"
+
+  bottle :unneeded
+  depends_on :java => "1.8"
+
+  def install
+    PROFILES.each do |profile|
+      bin.install "bin/#{PRODUCT_NAME}-#{PRODUCT_VERSION}-#{profile}"
+    end
+
+    libexec.install Dir["*"]
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.8"))
+  end
+
+  def caveats
+    caveat_msg = "Run the following commands to start each profile of the WSO2 Enterprise Integrator.\n"
```

Comment:
This isn't Homebrew specific information, it should be removed.

Reply:
Can you elaborate more on this issue? Do you mean that this information is wrong to put in the caveats?

Reply:
Yes.

Reply:
So if we need to give post-installation information, is there a place put it in Homebrew?

Reply:
That is the perview of upstream documentation.